azurerm/resource_arm_application_gateway.go

```diff
@@ -2645,6 +2712,37 @@ func flattenApplicationGatewayWafConfig(input *network.ApplicationGatewayWebAppl
 		output["rule_set_version"] = *input.RuleSetVersion
 	}
 
+	ruleGroups := make([]interface{}, 0)
+	if disabledRuleGroups := input.DisabledRuleGroups; disabledRuleGroups != nil {
+		for _, ruleGroup := range *disabledRuleGroups {
+			ruleGroupOutput := map[string]interface{}{}
+
+			if ruleGroup.RuleGroupName != nil {
+				ruleGroupOutput["rule_group_name"] = *ruleGroup.RuleGroupName
+			}
+
+			ruleOutputs := make([]interface{}, 0)
+			if rules := ruleGroup.Rules; rules != nil {
+				for _, rule := range *rules {
+					ruleOutputs = append(ruleOutputs, rule)
+				}
+			}
+			ruleGroupOutput["rules"] = ruleOutputs
+
+			ruleGroups = append(ruleGroups, ruleGroupOutput)
+		}
+
+		output["disabled_rule_group"] = ruleGroups
+	}
+
+	if input.RequestBodyCheck != nil {
```

Comment:
It looks like this was already set. https://github.com/terraform-providers/terraform-provider-azurerm/blob/3cc0aa473a4da0afa544f9cd7211a047fcbee651/azurerm/resource_arm_application_gateway.go#L2750 Was there a reason this was re-added here or can we remove this?